deployment/run_server

```diff
@@ -1,10 +1,25 @@
 #!/bin/bash -xe
 
 fc-cache -v /usr/share/fonts/user
+set -e
+
+BUILD_DIR=/var/www/image-exporter/build
+URL_REGEXP='(https?|ftp|file)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]'
+
+if [ ! -z ${PLOTLY_JS_SRC+x} ]; then
+  if [[ $PLOTLY_JS_SRC =~ $URL_REGEXP ]]; then
+    # Fetch plotly js bundle and save it locally:
+    mkdir -p $BUILD_DIR
+    wget --retry-connrefused --no-check-certificate -O $BUILD_DIR/plotly-bundle.js $PLOTLY_JS_SRC
+    PLOTLYJS_ARG="--PlotlyJS $BUILD_DIR/plotly-bundle.js"
+  else
+    # Pass on the exact arguments:
+    PLOTLYJS_ARG="--PlotlyJS $PLOTLY_JS_SRC"
```

Comment:
Note that the argument is `--plotlyJS`. Perhaps this works as well but please use the correct argument for consistency.